### ISSUE TITLE
/osio/import - PROD - Do we need GITHUB_CREATE and GITHUB_PUSHED status in the web socket response

### ISSUE THREAD

**Issue by jyasveer** [fabric8-launcher/launcher-planning#138]
Below is the request for the osio/import end point in prod
https://gist.github.com/jyasveer/df596df6f75a6825a064537014da3e15#file-osio-import-request

Here is the response 
```
{"uuid":"df541c56-c590-4360-a0b6-c59953628452","uuid_link":"/status/df541c56-c590-4360-a0b6-c59953628452"}
```

I see success notifications for all the 5 processes from the web socket response in the /osio/launch api call. But in the above api call i.e. `/osio/import` do we require notifications for `GITHUB_CREATE` and `GITHUB_PUSHED` steps.

**Comment by jyasveer** [comment]
@gastaldi @animuk can you please look into this ?

**Comment by gastaldi** [comment]
Does it work in prod-preview?

**Comment by gastaldi** [comment]
`GITHUB_CREATE` is never fired in /osio/import

**Comment by gastaldi** [comment]
We're not updating production until it works in prod-preview

**Comment by gastaldi** [comment]
Fixed through #167
